src/main/java/domain/car/Car.java

```diff
@@ -0,0 +1,38 @@
+package domain.car;
+
+import domain.name.Name;
+
+public class Car {
+    private static final int INITIAL_POSITION=0;
+    public static final int FORWARD_MAX_NUMBER = 10;
+    private final Name name;
+    private int position;
+    private static final int FORWARD_THRESHOLD = 4;
```

Comment:
```suggestion
    public static final int FORWARD_MAX_NUMBER = 10;
    private static final int INITIAL_POSITION=0;
    private static final int FORWARD_THRESHOLD = 4;
    private final Name name;
    private int position;
```
자바의 컨벤션은 `Class ( static) variables` - `Instance variables` 입니다.

https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html

Reply:
감사합니다!